pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "acapy_agent"
-version = "1.1.0"
+version = "1.1.1"
```

Comment:
Just a heads up that this probably should've been `1.1.1rc0`. Seems now `1.1.1` is already written to pypi (only matters if there's gonna be a diff between rc0 and 1.1.1

Reply:
Argghhhh…thanks for the heads up.  We’ll have to decide what to do — likely go to 1.1.2, since that version is now burned.

Doh.  Humans can be stoopid.  Especially me.